docs/_book.yaml

```diff
@@ -25,8 +25,10 @@ upper_tabs:
     #   skip_translation: true
     #   contents:
     #   - include: /tensorboard/api_docs/python/_toc.yaml
-    - name: TF 2.0 Alpha
+    - name: TF 2.0 Beta
```

Comment:
We should also update the button label on the subsite landing page here:
https://github.com/tensorflow/tensorboard/blob/abd11d6aac6203d2fd3fac11f70192ea0d141a70/docs/_index.yaml#L28